<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLBPControl</class>
 <widget class="QWidget" name="MLBPControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="paramMLLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="maxIterTextLabel">
       <property name="text">
        <string>Maximum iterations (EM):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="maxIterSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tolTextlabel">
       <property name="text">
        <string>Tolerance (EM):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="tolSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="iterTextLabel">
       <property name="text">
        <string>Iterations:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="iterSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="weightedCheckBox">
       <property name="text">
        <string>Weighted</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="llLabel">
       <property name="text">
        <string>Log-likelihood:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="llSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Time [s]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="timeSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="residualLabel">
       <property name="text">
        <string>Residual SSe:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="residualSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="paramBPLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="maxIterBPLabel">
       <property name="text">
        <string>Maximum iterations (BP):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="maxIterBPSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tolBPLabel">
       <property name="text">
        <string>Tolerance (BP):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="tolBPSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="runLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="runButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="showLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showMLBPCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show ML-BP segmentation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="testAction">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showMLBPCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MLBPControl</receiver>
   <slot>showMLBPToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MLBPControl</receiver>
   <slot>maxIterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tolSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MLBPControl</receiver>
   <slot>tolChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>pressed()</signal>
   <receiver>MLBPControl</receiver>
   <slot>runPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weightedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MLBPControl</receiver>
   <slot>weightedToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIterBPSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MLBPControl</receiver>
   <slot>maxIterBPChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tolBPSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MLBPControl</receiver>
   <slot>tolBPChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showMLBPToggled(bool)</slot>
  <slot>maxIterChanged(int)</slot>
  <slot>tolChanged(double)</slot>
  <slot>runPressed()</slot>
  <slot>weightedToggled(bool)</slot>
  <slot>maxIterBPChanged(int)</slot>
  <slot>tolBPChanged(double)</slot>
 </slots>
</ui>
